### ISSUE TITLE
hello,i am a rookie,i meet a problem

### ISSUE THREAD

**Issue by xzx-wh04412** [zhulf0804/UNet-Tensorflow#4]
i config the config.py to fit my own dataset,and the dataset is organized as you suggest.But when i python train.py,my Training loss is nan from the very first.i try to change learning rate to 0,and change batch size to 1,but training loss is still nan.
How can i sovle it？ 
![1582123651(1)](https://user-images.githubusercontent.com/61240096/74845623-4061a000-536a-11ea-9424-c6668337cf39.jpg)
![IMG_0698](https://user-images.githubusercontent.com/61240096/74845626-40fa3680-536a-11ea-8ea3-9f5f84567e3b.png)
![IMG_0698 (2)](https://user-images.githubusercontent.com/61240096/74845631-42c3fa00-536a-11ea-8ed1-235d19c4794c.png)
